src/components/Contentful/SectionWithBackgroundClassic.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+	<section class="">
+		<div class="tw-relative tw-p-0">
+			<div
+				class="tw-relative tw-w-full tw-overflow-hidden tw-z-1 tw-m-0 tw-top-0"
+				:class="verticalPaddingClasses"
+			>
+				<slot name="content">
+				</slot>
+			</div>
+			<div
+				class="tw-w-full tw-h-full tw-absolute tw-top-0 tw-z--1"
+				:style="backgroundStyle"
+			>
+				<video class="tw-w-full tw-h-full tw-object-cover" v-if="isBackgroundVideo"
+					:src="backgroundMedia.url"
+					autoplay
+					loop
+					muted
+					playsinline
+				></video>
+				<kv-contentful-img class="tw-w-full tw-h-full tw-object-cover section-with-background__media"
+					v-if="isBackgroundImage"
+					:width="1440"
+					:contentful-src="backgroundMedia.url"
+					fallback-format="jpg"
+					:alt="backgroundMedia.description"
+					:source-sizes="sourceSizes"
+				/>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+
+/**
+* Section Background
+* Creates a full width section with a full bleed background.
+* */
+
+export default {
+	components: {
+		KvContentfulImg,
+	},
+	props: {
+		/**
+		 * Content group content for background from Contentful type Background
+		* */
+		backgroundContent: {
+			type: Object,
+			default: () => {},
+		},
+		verticalPadding: {
```

Comment:
Using a UiSetting with a dataObject in Contentful to set values like so:
`{
    "verticalPadding": {
        "sm": {
            "top": 6,
            "bottom": 8
        },
        "md": {
            "top": 0,
            "bottom": 0
        },
        "xl": {
            "top": 3,
            "bottom": 0
        }
    }
}`

Reply:
I think in a subsequent version we'll switch this to more generic words such as `none`, `small`, `medium`, `large`, `xlarge`...